### ISSUE TITLE
remember settings?

### ISSUE THREAD

**Issue by richo67** [UniverseMaker/EyeDrive#1]
This is great simple thing. Could it remember the setting I type once? Next time when I started I have to typed them again. Also chache seems never be clened up. After each connect new folder is made.

**Comment by UniverseMaker** [comment]
Sorry i pressed wrong button. :) I will update for information saving function also i will check bug for cache clean up. thank you for apply issues.

**Comment by UniverseMaker** [comment]
@richo67 When you connect to server create what type of directory? some directory? or local drive? Could you upload some screenshot for debugging?

**Comment by richo67** [comment]
I am connecting to webdav server on synology and mounting it as drive (I mean this is what your dialog box does with help of rclone. In cache folders are created each time I run it with different name.

BTW: You do not have to worry about it. I am using now rclone directly. I have added option --volname (or is it --volumename) which makes sure that folde rin cache has same name all the time (after starting again) and also the name of the mounted drive (volume name) in windows is same.

**Comment by UniverseMaker** [comment]
@richo67 Thank you for your reply. I will add volname option.

**Comment by richo67** [comment]
That may help, but what I mean originally was the fact to save all the options, once typed to the edit fields.
If once I typed them and I disconnect the mount and close your program, it would be nice to have all the options back once I start it again. 
What do you think? Is it something useful?

**Comment by UniverseMaker** [comment]
@richo67 Yes, that function will be very useful :). I will update saving function today. But i was designed this software for removing all connection information when using the rclone. Because rclone software automatically saving all information for connect. It is very very risky software for research area. this area require all software saving nothing :0 haha. So i designed this GUI software.

So when i add saving function. It is can use optional. You can turn it on or turn it off. :)

**Comment by richo67** [comment]
Making it optional, sort of check box (save yes/no) in the dialog itself, is a perfect approach. Thank you.

**Comment by UniverseMaker** [comment]
Application Updated. Thank you for your contribution. :)
